src/main/src/main/java/org/geoserver/config/GeoServerPropertyConfigurer.java

```diff
@@ -114,4 +114,12 @@ protected void loadProperties(Properties props) throws IOException {
             }
         }
     }
+    
+    public void reload() throws IOException {
```

Comment:
Answering myself, yes, by the new backup/restore module.
New public method, please add javadoc